zubhub_frontend/zubhub/src/assets/js/styles/components/project/projectStyles.js

```diff
@@ -118,13 +120,23 @@ const styles = theme => ({
     marginTop: '0.5em',
     marginBottom: '0.5em',
     display: 'flex',
-    justifyContent: 'flex-start',
     alignItems: 'center',
+    width: '100%',
   },
   creatorAvatarStyle: {
     marginRight: '0.5em',
     boxShadow: `0 3px 5px 2px rgba(0, 0, 0, .12)`,
   },
+  creatorEmailAndTagBoxStyle: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    [theme.breakpoints.down('376')]: {
```

Comment:
Its better to use predefined breakpoints like sm or md 

https://mui.com/material-ui/customization/breakpoints/

Reply:
with the pre-defined breakpoints, the values for the sm or md are quite high so in order to get the wrap to work without changes, I created the box and used that style just for that line.

Reply:
ok @kamthamc @tuxology  i'll do that now and push

Reply:
If we want to use custom size we could define a different size while creating the theme and use it here